code/datums/datumvars.dm

```diff
@@ -929,6 +929,24 @@
 			manipulate_organs(C)
 			href_list["datumrefresh"] = href_list["editorgans"]
 
+		else if(href_list["hallucinate"])
+			if(!check_rights(0))
+				return
+
+			var/mob/living/carbon/C = locate(href_list["hallucinate"])

```

Comment:
this is datum editing lzimann, these verbs are only accessible to admins so it doesn't matter as much, they can already locate any object in the world, scoping locates only matters when players can access the topic, since they can then pass in their own ref

you could do locate(thing) in GLOB.mob_list to scope it though